.circleci/config.yml

```diff
@@ -4,11 +4,11 @@ executors:
   basic-executor:
     resource_class: small
     docker:
-      - image: cimg/ruby:3.2.2-browsers
```

Comment:
Is the ruby upgrade necessarily bundled in this ticket?

Reply:
Sorry, not sure what you mean?

Reply:
Yes, was required for this approach:

> bumping version of ruby and the alpine image (ruby:3.2.3-alpine3.19) to get the it’s latest version of Chromium, which would hopefully be compatible to with puppeteer@21.9.0 or puppeteer@22.2.0. This was not possible, and came across errors in UAT when trying to download a PDF.

Reply:
Yes, exactly that.